index.html

```diff
@@ -5,6 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Soul Shoes</title>
+    <style>
+        body {
+            background-color: red;
```

Comment:
This really looks pretty bad.  Maybe something lighter?

Reply:
Whatever

Reply:
I've updated it to a light red. Does this please you NOW?

Reply:
Way better!!!  Love it!